docs/sparkr.md

```diff
@@ -262,6 +262,83 @@ head(df)
 {% endhighlight %}
 </div>
 
+### Applying User-defined Function
+In SparkR, we support several kinds for User-defined Functions:
+
+#### Run a given function on a large dataset using `dapply` or `dapplyCollect`
+
+##### dapply
+Apply a function to each partition of `SparkDataFrame`. The function to be applied to each partition of the `SparkDataFrame`
```

Comment:
@NarineK I filed a JIRA [SPARK-16112](https://issues.apache.org/jira/browse/SPARK-16112) for gapply programming guide so that you could open a PR for this.